test/path.jl

```diff
@@ -85,6 +85,38 @@
     end
     @test relpath(S(joinpath("foo","bar")), S("foo")) == "bar"
 
+    @testset "splitpath" begin
+        @test ["a", "b", "c"] == splitpath(joinpath("a","b","c"))
```

Comment:
Ordering the test here and below as `@test splitpath(joinpath("a","b","c")) == ["a", "b", "c"]` is clearer IMO.

Reply:
Done, thanks! I should've noticed that the surrounding test style is the way you have it. Old habits from my last company. :) Thanks!